### PR TITLE
Add two.

### PR DESCRIPTION
**Pull request by bangerth**

### CODE REVIEW

File: publications-2022.bib [hunk 1]
```diff
@@ -1003,6 +1003,17 @@ @Manual{2022:vidal-ferrandiz.carreno.ea:femffusion
   url     = {https://femffusion.webs.upv.es/wp-content/uploads/2022/02/FEMFFUSION_Manual_es.pdf}
 }
 
+@Article{2022:wang.wang.ea:robust,
+  author  = { Wang, Zhuoran and Wang, Ruishu and Liu, Jiangguo },
+  title   = { Robust weak Galerkin finite element solvers for Stokes flow based on a lifting operator },
+  journal = { Computers \&amp; Mathematics with Applications },
```
Comment (marcfehling) [inline publications-2022.bib:1009]:
```suggestion
  journal = { Computers \& Mathematics with Applications },
```

**Review by marcfehling — APPROVED** [review]
Looks good, with one suggestion

**Comment by bangerth** [comment]
Good point. Let's try this now.

**Pull request merged**